cmake/serac-config.cmake.in

```diff
@@ -31,6 +31,7 @@ if(NOT SERAC_FOUND)
 
   set(SERAC_USE_ADIAK          @SERAC_USE_ADIAK@)
   set(SERAC_USE_AXOM           @SERAC_USE_AXOM@)
+  set(SERAC_USE_CAMP           TRUE)#@SERAC_USE_CAMP@)
```

Comment:
This can be un-hardcoded by adding camp to this list:

https://github.com/LLNL/serac/blob/1783d8102e7aa0aa9ed41f6d345548029346eec6/cmake/SeracConfigHeader.cmake#L39